pkg/clients/metadata/device_profile.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/edgexfoundry/edgex-go/pkg/clients"
 	"github.com/edgexfoundry/edgex-go/pkg/clients/types"
 	"github.com/edgexfoundry/edgex-go/pkg/models"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"path/filepath"
```

Comment:
@weichou1229 Please put this block of imports above the github.com import list and alphabetize. Rebase your feature branch to clear the "out-of-date" message. After that, should be good.

Reply:
`go fmt` should do this for you, please set-up your editor to use it.